locales/region/fr.yml

```diff
@@ -1759,461 +1759,461 @@ nis-64015E: Fumal (Braives)
-nis-63038: Jalhay + submsous-municipalit\u00e9s
+nis-63035H: Julémont (Herve)
+nis-63038: Jalhay + sous-municipalités
 nis-63038A: Jalhay (Jalhay)
 nis-63038B: Sart (Jalhay)
-nis-63040: Kelmis + submsous-municipalit\u00e9s
-nis-63040A: Kelmis (Kelmis)
-nis-63040B: Neu-Moresnet (Kelmis)
-nis-63040C: Hergenrath (Kelmis)
-nis-63045: Lierneux + submsous-municipalit\u00e9s
+nis-63040: La Calamine 'D' + sous-municipalités
+nis-63040A: La Calamine 'D' (La Calamine 'D')
+nis-63040B: Neu-Moresnet (La Calamine 'D')
+nis-63040C: Hergenrath (La Calamine 'D')
+nis-63045: Lierneux + sous-municipalités
 nis-63045B: Bra (Lierneux)
 nis-63045C: Arbrefontaine (Lierneux)
 nis-63045A: Lierneux (Lierneux)
-nis-63046: Limburg + submsous-municipalit\u00e9s
-nis-63046A: Limbourg (Limburg)
-nis-63046B: Bilstain (Limburg)
-nis-63046C: Go\u00e9 (Limburg)
-nis-63048: Lontzen + submsous-municipalit\u00e9s
+nis-63046: Limbourg + sous-municipalités
+nis-63046A: Limbourg (Limbourg)
+nis-63046B: Bilstain (Limbourg)
+nis-63046C: Goé (Limbourg)
+nis-63048: Lontzen + sous-municipalités
 nis-63048A: Lontzen (Lontzen)
 nis-63048B: Walhorn (Lontzen)
-nis-63049: Malmedy + submsous-municipalit\u00e9s
-nis-63049A: Malmedy (Malmedy)
-nis-63049C: Bellevaux-Ligneuville (Malmedy)
-nis-63049B: Beverc\u00e9 (Malmedy)
-nis-63057: Olne + submsous-municipalit\u00e9s
+nis-63049: Malmédy + sous-municipalités
+nis-63049A: Malmédy (Malmédy)
+nis-63049C: Bellevaux-Ligneuville (Malmédy)
+nis-63049B: Bevercé (Malmédy)
+nis-63057: Olne + sous-municipalités
 nis-63057A: Olne (Olne)
-nis-63058: Pepinster + submsous-municipalit\u00e9s
+nis-63058: Pepinster + sous-municipalités
 nis-63058A-II: Cornesse (Pepinster)
 nis-63058C: Wegnez (Pepinster)
 nis-63058B: Soiron (Pepinster)
 nis-63058A-I: Pepinster (Pepinster)
-nis-63088: Plombi\u00e8res + submsous-municipalit\u00e9s
-nis-63088A: Gemmenich (Plombi\u00e8res)
-nis-63088B: Moresnet (Plombi\u00e8res)
-nis-63088C: Montzen (Plombi\u00e8res)
-nis-63088D: Hombourg (Plombi\u00e8res)
-nis-63088E: Sippenaeken (Plombi\u00e8res)
-nis-63061: Raeren + submsous-municipalit\u00e9s
+nis-63088: Plombières + sous-municipalités
+nis-63088A: Gemmenich (Plombières)
+nis-63088B: Moresnet (Plombières)
+nis-63088C: Montzen (Plombières)
+nis-63088D: Hombourg (Plombières)
+nis-63088E: Sippenaeken (Plombières)
+nis-63061: Raeren + sous-municipalités
 nis-63061A: Raeren (Raeren)
 nis-63061B: Eynatten (Raeren)
 nis-63061C: Hauset (Raeren)
-nis-63067: Sankt Vith + submsous-municipalit\u00e9s
-nis-63067-Z: Sankt Vith (Sankt Vith)
-nis-63067D: Lommersweiler (Sankt Vith)
-nis-63067E: Crombach (Sankt Vith)
-nis-63067F: Recht (Sankt Vith)
-nis-63067C: Sch\u00f6nberg (Sankt Vith)
-nis-63072: Spa + submsous-municipalit\u00e9s
+nis-63067: Saint-Vith 'D' + sous-municipalités
+nis-63067-Z: Saint-Vith 'D' (Saint-Vith 'D')
+nis-63067D: Lommersweiler (Saint-Vith 'D')
+nis-63067E: Crombach (Saint-Vith 'D')
+nis-63067F: Recht (Saint-Vith 'D')
+nis-63067C: Schönberg (Saint-Vith 'D')
```

Comment:
What's this? [https://fr.wikipedia.org/wiki/Saint-Vith](https://fr.wikipedia.org/wiki/Saint-Vith)